src/js/hyperschedule.js

```diff
@@ -343,6 +349,13 @@ function courseToInstructorLastnames(course)
   return course.courseInstructors.map(fullName => fullName.split(",")[0]).join(",");
 }
 
+function coursesEqual(course1, course2) {
+  if (course1.courseCode == course2.courseCode) {
+    return true;
+  }
+  return false;
```

Comment:
[We should never use `==`.](https://2ality.com/2011/12/strict-equality-exemptions.html) I would also suggest just refactoring this whole function to `return course1.courseCode === course2.courseCode`.

Reply:
This should be resolved in the newest commit.